### ISSUE TITLE
Narrator does not read items properly in List component

### ISSUE THREAD

**Issue by avenezia** [OfficeDev/office-ui-fabric-react#11078]
### Environment Information

- **Package version(s)**: 7.40.0
- **Browser and OS versions**: Edge 44.18362.387.0

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:
Focusing the first list item in the codepen, Narrator reads "group", and it does not read any information about the list item. I see that in the _onRenderCell method of the codepen, the main div has the data-is-focusable={true} attribute; if I use the DevTools to remove it (together with the tabindex) and set the data-is-focusable={true} in the wrapping div (the one with class ms-List-cell and listitem role), Narrator reads "Lorem ipsum..." and as well the list item position (e.g. 1 of 30 level 1). Do you think that allowing the focus on the cell div would be better?

Furthermore, the list in the codepen contains 5000 items but, even with the change I mentioned above, Narrator is not able to determine the right amount of items: shouldn't the list items include an aria-posinset attribute, especially in a virtualized list?

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/avenezia/pen/NWWPomp (it corresponds to the "Rendering ghost items while the list is scrolling" codepen in the UI Fabric Basic List [page](https://developer.microsoft.com/en-us/fabric#/controls/web/list)

#### Actual behavior:

List item information is not read by Narrator

#### Expected behavior:

Narrator should read list item information.

### Documentation describing expected behavior

https://www.w3.org/TR/wai-aria-1.1/#listitem

**Comment by JasonGore** [comment]
The [ARIA `list` role])(https://www.w3.org/TR/wai-aria-1.1/#list) does not have `aria-row` attributes, and therefore would not communicate row information. However, the `role` is available via `IList` props and can be changed to support the scenario you are looking for. Additional properties such as `aria-rowcount` and `aria-row` can be applied directly on `List` and in `onRenderCell`, as well.

I see your observation about `data-is-focusable`. Oddly enough it seems to work as expected in Chrome. However, this seems to be limited to the example code and not the `List` implementation itself.

In general we consider `List` to have so many possible use cases that we can't know the correct accessibility behavior. Rather it is up to consumers of `List` to make `List` accessible for their needs, such as narrating row information.

**Comment by avenezia** [comment]
Thanks for you answer! 
In conclusion, due to the different use cases that List has to support, it is better to not put too much accessibility logic in the List implementation, but rather let the user customize it in the onRenderCell.

**Comment by JasonGore** [comment]
Since I first commented I came across #10939 which is possibly a dupe. I was incorrect in my initial comment. The `listitem` role does support positioning information and attributes (such as `aria-posinset`) but it appears that [`list` role behaviors very inconsistently among browser and screen reader combinations](https://www.scottohara.me/blog/2018/05/26/aria-lists.html). If that changes your view on this issue, please let me know. For now I'll leave this closed and consider it a duplicate of #10939.
